docs/validators/emergency-upgrades.md

```diff
@@ -0,0 +1,83 @@
+---
+order: 5
+title: Emergency Upgrades
+---
+
+## **Introduction**
+
+This article goes into the general flow of events for an emergency upgrade, pre-upgrade and during the upgrade for all validators.
+
+Emergency upgrades are an unfortunate necessity for all validators when running a PoS node.
+
+When a vunerability is discovered and responsibily disclosed, the vunerability will be reported to one of the channels described in the [Security.md](../../SECURITY.md).
```

Comment:
has there been coordination about this with greg or someone from amulet? want to make sure that these processes are all aligned.